Cargo.lock

```diff
@@ -1800,7 +1800,8 @@ dependencies = [
  "security-framework",
  "smoltcp",
  "socket2",
- "sysinfo",
+ "sysinfo 0.29.11",
+ "sysinfo 0.33.0",
```

Comment:
What's up with the sysinfo version for macOS? Can we update that in a seperate PR?